ote/src/cljs/ote/views/ckan_org_viewer.cljs

```diff
@@ -42,19 +42,20 @@
 (def contact-methods [::to-definitions/phone
                       ::to-definitions/gsm
                       ::to-definitions/email
-                      ::to-definitions/facebook
-                      ::to-definitions/twitter
-                      ::to-definitions/instant-message])
+                     ; ::to-definitions/facebook
```

Comment:
Remove these lines if we are not going to support social media contact methods.

Reply:
removed